### PR TITLE
Feature: Search List Container: Search Input , Reducers and Selectors added

### PR DESCRIPTION
**Pull request by rituraj-wednesday**
Search List Container created:
- Title, description and message texts
- Search bar
- Reducer, Selector and Saga files
- Unit tests for each files

Component:
SearchListContainer

### CODE REVIEW

File: app/containers/SearchListContainer/index.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+import saga from './saga';
+import { translate } from '@app/utils';
+
+const CustomCard = styled(Card)`
+  && {
+    margin: 1.25rem 0;
+    padding: 1rem;
+    max-width: ${(props) => props.maxwidth};
+    color: ${(props) => props.color};
+    ${(props) => props.color && `color: ${props.color}`};
```
Comment (rajatdangat-ws) [inline app/containers/SearchListContainer/index.js]: Why do we need to set the color twice?

File: app/containers/SearchListContainer/reducer.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+export const initialState = {
+  term: null,
+  error: null,
+  loading: false
+};
+
+export const { Types: searchListContainerTypes, Creators: searchListContainerCreators } = createActions({
+  requestGetSearchedTunes: ['term'],
+  successGEtSearchedTunes: ['data'],
```
Comment (rajatdangat-ws) [inline app/containers/SearchListContainer/reducer.js]: There is a typo on this line

File: grep
Comment (rajatdangat-ws) [inline grep:1]: Why do we need this file?

Comment (rituraj-wednesday) [reply grep:1]: It was due to a bad command. It was unnecessary, I have removed it.

File: app/containers/SearchListContainer/index.js (hunk 1 above)
Comment (rituraj-wednesday) [reply app/containers/SearchListContainer/index.js]: removed an unnecessary line. I think it was added while removing lint error.

File: app/containers/SearchListContainer/reducer.js (hunk 2 above)
Comment (rituraj-wednesday) [reply app/containers/SearchListContainer/reducer.js]: fixed the typo

**Review by rajatdangat-ws — APPROVED** [review]

**Pull request merged**